libraries/abstractions/secure_sockets/lwip/iot_secure_sockets.c

```diff
@@ -114,6 +115,19 @@ static int8_t sockets_allocated = socketsconfigDEFAULT_MAX_NUM_SECURE_SOCKETS;
 
 /*-----------------------------------------------------------*/
 
+#if AFR_ESP_LWIP
```

Comment:
@gkwicker Thanks for your input, I thought about creating platform specific copy but due to minimal changes I was bit reluctant. It seems bit difficult to find any other way to redirect secure socket layer to use VFS, hence falling back to use platform specific layer instead, have updated PR to use `./vendors/espressif/boards/esp32/ports/secure_sockets/lwip/iot_secure_sockets.c`

For second point, yes fixes are generic (IMO). You may absorb them in case you are facing issue with AFQP run for `FullSecureSockets` module for other platforms.